labgrid/remote/client.py

```diff
@@ -1568,6 +1626,16 @@ def main():
     subparser.add_argument('mountpoint', help='local path')
     subparser.set_defaults(func=ClientSession.sshfs)
 
+    subparser = subparsers.add_parser('forward',
+                                      help="forward local port to remote target")
+    subparser.add_argument("--local", "-L", metavar="[LOCAL:]REMOTE",
+                           action=LocalPort,
+                           help="Forward local port LOCAL to remote port REMOTE. If LOCAL is unspecified, an arbitrary port will be chosen")
+    subparser.add_argument("--remote", "-R", metavar="REMOTE[:LOCAL]",
+                           action=RemotePort,
+                           help="Forward remote port REMOTE to local port LOCAL. If LOCAL is unspecified, an arbitrary port will be chosen")
```

Comment:
When is the arbitrary port feature useful for the command line client?

Also, I think that in the -R case, the LOCAL (target) needs to be explicit, otherwise you wouldn't know what it connects to (see my comment to `forward_remote_port()` in the driver).